index.js

```diff
@@ -127,6 +177,27 @@ client.on(Events.MessageReactionAdd, async (reaction) => {
         // Check the reaction emoji for thread archival confirmation
         if (emoji === '👍') {
             parentChannel.send(`You've confirmed the archival of the thread`);
+
+            // Create message link
+            const messageLink = `https://discord.com/channels/${reaction.message.guildId}/${reaction.message.channelId}/${reaction.message.id}`;
+
+            const processingTask = `Process ${ARCHIVED_THREAD.name} in Monolith: ${messageLink} #"⏳ Productivity" +today ~10m @Processing`;
```

Comment:
Hey I have a small request - do you think we could set the processing task category name (e.g. `#"⏳ Productivity`) separately per user? Right now all my tasks go to the Inbox since the category doesn't match on my end. Not urgent at all, but would be a big quality-of-life boost if it's an easy change. Appreciate it!